codex-rs/tui/src/cli.rs

```diff
@@ -17,6 +17,10 @@ pub struct Cli {
     #[arg(long, short = 'm')]
     pub model: Option<String>,
 
+    /// Service tier for request, defaults to auto, use flex to enable Flex Processing.
+    #[arg(long)]
+    pub service_tier: Option<String>,
+
```

Comment:
```suggestion
```
I would kinda expect rust fmt to take care of these

Reply:
I don't think I got that working yet, need to set up their pre-commit thing

Reply:
This one doesn't appear to be a rust fmt error. Not sure there needs to be spacing here?

Reply:
I could be wrong!